types/priv_validator.go

```diff
@@ -73,7 +83,7 @@ func (pv MockPV) GetPubKey() (crypto.PubKey, error) {
 func (pv MockPV) SignVote(chainID string, vote *cmtproto.Vote) error {
 	useChainID := chainID
 	if pv.breakVoteSigning {
-		useChainID = "incorrect-chain-id"
+		useChainID = MockChainID
```

Comment:
I didn't want to change this, but the linter decided it cared about this

Reply:
yes linter is selective on files we change for some reason